patterns/pages/all_announcements_page.php

```diff
@@ -1,428 +1,378 @@
-
-                    <!-- wp:post-title {"isLink":true,"linkTarget":"_blank","style":{"typography":{"fontSize":"18px","fontStyle":"normal","fontWeight":"600","letterSpacing":"-0.18px"},"color":{"text":"#002145"}}} /-->
-
-                    <!-- wp:post-excerpt {"style":{"typography":{"fontSize":"16px"},"color":{"text":"#002145"}}} /-->
-                    <!-- /wp:post-template -->
-
-                    <!-- wp:query-pagination {"align":"center","className":"mednet_pagination_block"} -->
-                    <!-- wp:query-pagination-previous /-->
-
-                    <!-- wp:query-pagination-numbers /-->
-
-                    <!-- wp:query-pagination-next /-->
-                    <!-- /wp:query-pagination -->
-
-                    <!-- wp:query-no-results -->
-                    <!-- wp:paragraph {"placeholder":"Add text or blocks that will display when the query returns no results."} -->
-                    <p>No results found.</p>
-                    <!-- /wp:paragraph -->
-                    <!-- /wp:query-no-results -->
-                </div>
-                <!-- /wp:query -->
-            </div>
-        </section>
-        <!-- /wp:ubc/tab -->
+      </li>
+    </ul>
+    <!-- wp:ubc/tab {"index":0,"title":"All Announcements"} -->
+    <section
+      index="1"
+      class="wp-block-ubc-tab ubc-accordion-tabs__tabs-panel js-tabs-panel active"
+      id="section1"
+      role="tabpanel"
+      aria-labelledby="tab1"
+    >
+      <div
+        class="ubc-accordion-tabs__accordion-trigger js-accordeon-trigger"
+        aria-controls="section1"
+        tabindex="0"
+      >
+        All Announcements
+        <div class="ubc-accordion-tabs__accordion-trigger-icon">
+          <span class="label--open">Open</span
+          ><span class="label--close">Close</span
+          ><svg aria-hidden="true" focusable="false" viewBox="0 0 20 20">
+            <rect
+              class="vert"
+              height="18"
+              width="2"
+              fill="currentColor"
+              y="1"
+              x="9"
+            ></rect>
+            <rect height="2" width="18" fill="currentColor" y="9" x="1"></rect>
+          </svg>
+        </div>
+      </div>
+      <div class="content">
+        <!-- wp:query {"queryId":33,"query":{"perPage":"3","pages":"0","offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"taxQuery":{"category":[5]}},"displayLayout":{"type":"flex","columns":3},"className":"announcement_feed_small"} -->
+        <div class="wp-block-query announcement_feed_small">
+          <!-- wp:post-template -->
+          <!-- wp:post-featured-image {"isLink":true,"align":"wide"} /-->
+
+          <!-- wp:post-title {"isLink":true,"linkTarget":"_blank","style":{"typography":{"fontSize":"18px","fontStyle":"normal","fontWeight":"600","letterSpacing":"-0.18px"},"color":{"text":"#002145"}}} /-->
```

Comment:
Remove target=_blank from queries.
Move styles to a variation if needed. Otherwise define for the site.